dist/src/main/config/application.yaml

```diff
@@ -134,7 +134,7 @@ camunda:
     # Zeebe instance
     zeebe:
       # Gateway address
-      gatewayAdress: localhost:26500
+      gatewayAddress: localhost:26500
```

Comment:
I'm not mistaken this config was not effective due to the typo, right? As the underlying properties class defined it as `gatewayAddress` already.

Reply:
ok, it worked because the default value is identical https://github.com/camunda/camunda/blob/main/tasklist/common/src/main/java/io/camunda/tasklist/property/ZeebeProperties.java#L12